pages/common/xml-list.md

```diff
@@ -0,0 +1,16 @@
+# xml-list
+
+> List a directory as XML.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- List the current directory to an XML file:
+
+`xml list >{{path/to/dir_list.xml}}`
+
+- List the specified directory to an XML file:
```

Comment:
Perhaps this will read better?
```suggestion
- List the items in the specified directory as an XML document:
```

Reply:
- List the specified directory's contents to an XML document: